cloud-operator/templates/cloud-operator-deployment.yaml

```
@@ -88,6 +88,8 @@ spec:
               value: "{{ .Values.env.streamSuccessPeriods.connect }}"
             - name: STREAM_SUCCESS_PERIOD_AUTH
               value: "{{ .Values.env.streamSuccessPeriods.auth }}"
+            - name: HTTPS_PROXY
+              value: "{{ .Values.httpsProxy }}"
```

Move this under `env` to be consistent?
```suggestion
              value: "{{ .Values.env.httpsProxy }}"
```